docs/ovms_quickstart.md

```diff
@@ -0,0 +1,170 @@
+│       └── ir_model.xml
+└── model2
+    └── 1
+        ├── ir_model.bin
+        ├── ir_model.xml
+        └── mapping_config.json
+``` 
+
+Each model should be stored in a dedicated folder (model1 and model2 in the examples above) and should include subfolders
+representing its versions. The versions and the subfolder names should be positive integer values. 
+
+Every version folder _must_ include a pair of model files with .bin and .xml extensions; however, the filename can be arbitrary.
+
+### Starting Model Server Container with a Single Model
+
+When models are ready and stored in the correct directory structure, you are ready to start the Docker container with the Model Server. To serve just a single model, no configuration file is required, just a single command:
+
+```docker run --rm -d  -v /models/:/opt/ml:ro -p 9001:9001 -p 8001:8001 ie-serving-py:latest \
+/ie-serving-py/start_server.sh ie_serving model --model_path /opt/ml/model1 --model_name my_model --port 9001 --rest_port 8001
+```
+
+Where:
+* option `-v` defines how the models folder should be mounted inside the docker container.
+* option `-p` exposes the model serving port outside the docker container.
+* `ie-serving-py:latest` represent the name of the image with OpenVINO&trade; Model Server. 
+* `start_server.sh` script activates the python virtual environment inside the docker container.
+* `ie_serving` command starts the model server with the following parameters:
+	* `--model_path` - path to the folder's structure with models
+	* `--mode_name` - name of the model
+	* `--port` - port on which gRpc service listens 
+	* `--rest_port` - port on which REST service listens
+
+### Starting Model Server Container with Multiple Models
+
+Model Server can serve multiple models at once. To start in this mode, a configuration file with the location of models is required. An example of this file is included below:
+
+```json
+{
+   "model_config_list":[
+      {
+         "config":{
+            "name":"model_name1",
+            "base_path":"/opt/ml/models/model1",
+            "batch_size": "16"
+         }
+      },
+      {
+         "config":{
+            "name":"model_name2",
+            "base_path":"/opt/ml/models/model2",
+            "batch_size": "auto",
+            "model_version_policy": {"all": {}}
+         }
+      }
+   ]
+}
+
+```
+The file is in JSON format and contains a mandatory array `model_config_list`, which includes a collection of `config` objects for each served model. Each config object includes values for the model `name` and the `base_path` attributes.
+
+To run Model Server in this mode - please run the following command:
+
+```docker run --rm -d  -v /models/:/opt/ml:ro -p 9001:9001 -p 8001:8001 ie-serving-py:latest \
+/ie-serving-py/start_server.sh ie_serving config config /opt/ml/config.json --port 9001 --rest_port 8001
+```
+
+**Note** do not forget to copy the configuration file to the `/models/` directory.
+
+### Starting Model Server Locally

```

Comment:
I think the term Locally can be confusing because docker container can be also local. Maybe better term could be like running directly on baremetal or VM?

Reply:
I've changed it